app/controllers/api_controller/parser/request_adapter.rb

```diff
@@ -39,15 +41,16 @@ def c_suffix
       end
 
       def c_id
-        @params[:c_id] || c_path_parts[1]
+        id = @params[:c_id] || c_path_parts[1]
```

Comment:
Ahh ok, and probably /api/settings too.  btw, we might have to do the same in s_id below once /api/automate is introduced in a later PR.   Thanks.